release: v1.43.0 (#4821)
apollographql/router: .changesets/feat_subscription_websocket_heartbeat.md

> ### Subscriptions: Add configurable "heartbeat" to subgraph WebSocket protocol ([Issue #4621](https://github.com/apollographql/router/issues/4621))
> 
> To account for GraphQL Subscription WebSocket implementations (e.g., [DGS](https://netflix.github.io/dgs/)) which drop idle connections by design, the router adds the ability to configure a heartbeat to keep active connections alive.
> 
> An example of configuration:
> 
> ```yaml
> subscription:
>   mode:
>     passthrough:
>       all:
>         path: /graphql
>         heartbeat_interval: enable #Optional

Reviewer: ```suggestion
### Support configurable heartbeat for subscriptions using the WebSocket protocol  ([Issue #4621](https://github.com/apollographql/router/issues/4621))

To support GraphQL Subscription WebSocket implementations such as [DGS](https://netflix.github.io/dgs/) that drop idle connections by design, the router adds the ability to configure a heartbeat to keep active connections alive.

An example router configuration:

```yaml 
subscription:
  mode:
    passthrough:
      all:
        path: /graphql
        heartbeat_interval: enable #Optional
```
